Description: Delay autostart of the gdu daemon for 10 seconds. It is not needed immediately (thus no need to slow down initial session startup) and is not visible permanently (thus it won't cause visual disruptions later on). (This depends on the patch in GNOME #608402, and will be a no-op if that's not available.)
Author: Martin Pitt <martin.pitt@ubuntu.com>
Origin: Ubuntu
Index: gnome-disk-utility/data/gdu-notification-daemon.desktop.in.in.in
===================================================================
--- gnome-disk-utility.orig/data/gdu-notification-daemon.desktop.in.in.in	2009-10-24 18:08:22.000000000 +0200
+++ gnome-disk-utility/data/gdu-notification-daemon.desktop.in.in.in	2010-02-23 23:46:21.000000000 +0100
@@ -12,3 +12,4 @@
 X-GNOME-Bugzilla-Component=notifications
 X-GNOME-Bugzilla-Version=@VERSION@
 #X-GNOME-AutoRestart=true
+X-GNOME-Autostart-Delay=10
